web/angular-wallet/src/app/main/settings/settings.component.ts

```diff
@@ -0,0 +1,128 @@
+import {Component, OnInit} from '@angular/core';
+import {FormControl} from '@angular/forms';
+import semver from 'semver';
+import {Settings} from '../../settings';
+import {constants} from '../../constants';
+import {environment} from '../../../environments/environment.hmr';
+import {I18nService} from '../../layout/components/i18n/i18n.service';
+import {StoreService} from '../../store/store.service';
+import {ActivatedRoute} from '@angular/router';
+import {ApiComposer, BurstService, getBlockchainStatus, getServerStatus} from '@burstjs/core';
+import {NotifierService} from 'angular-notifier';
+
+interface NodeInformation {
+  url: string;
+  version: string;
+}
+
+const UnsupportedFeatures = {
+  [constants.multiOutMinVersion]: 'node_hint_no_multiout',
+};
+
+@Component({
+  selector: 'app-settings',
+  templateUrl: './settings.component.html',
+  styleUrls: ['./settings.component.scss']
+})
+export class SettingsComponent implements OnInit {
+
+  constructor(private i18nService: I18nService,
+              private storeService: StoreService,
+              private notifierService: NotifierService,
+              private route: ActivatedRoute) {
+  }
+
+  public selectedNode = new FormControl();
+  public settings: Settings;
+
+  public nodes = SettingsComponent.createNodeList();
+  public isFetchingNodeInfo = false;
+
+  private static createNodeList(): Array<NodeInformation> {
+    return constants.nodes.map(({address, port}) => ({
+        url: `${address}:${port}`,
+        version: null,
+      })
+    ).concat({
+      url: environment.defaultNode,
+      version: null,
+    });
+  }
+
+  static async fetchNodeInformation(url: string): Promise<NodeInformation> {
+    try {
+      const networkApi = ApiComposer
+        .create(
+          new BurstService({
+            nodeHost: url,
+            apiRootUrl: '/burst',
+          }))
+        .withNetworkApi({
+          getBlockchainStatus,
+        })
+        .compose();
+      const {version} = await networkApi.network.getBlockchainStatus();
+      return {
+        url,
+        version,
+      };
+    } catch (e) {
+      // no op
```

Comment:
If it fails to get the version, do we need to account for that?

Reply:
It's handled in Line 97 in this file. If no info can be fetched, the targeted node won't be selectable then.